python/doc/examples/reliability_sensitivity/reliability/plot_crossentropy.py

```diff
@@ -0,0 +1,328 @@
+"""
+Cross Entropy Importance Sampling
+=================================
+"""
+# %%
+#
+# The objective is to evaluate a failure probability from the two versions of Cross Entropy Importance Sampling working in Standard or Physical spaces, see :class:`~openturns.StandardSpaceCrossEntropyImportanceSampling` and :class:`~openturns.PhysicalSpaceCrossEntropyImportanceSampling`.
+# We consider the simple stress beam example: :ref:`axial stressed beam <use-case-stressed-beam>`.
+
+
+# %%
+# First, we import the python modules:
+
+# %%
+import openturns as ot
+import openturns.experimental as otexp
+import openturns.viewer as viewer
```

Comment:
I suggest to use `otv` as is presented in the help page of the [Viewer](http://openturns.github.io/openturns/latest/user_manual/_generated/openturns.viewer.View.html#openturns.viewer.View). This is because most modules are short-names with 2 (numpy as np, openturns as ot) or 3 letters. The name `viewer` may be confused with `view` with is the name used on output of the `View` class.

```suggestion
import openturns.viewer as otv
```

Reply:
Fixed !